src/components/OrdersWidget/index.tsx

```diff
@@ -191,14 +192,31 @@ const OrdersWidget: React.FC = () => {
     [orders, setMarkedForDeletion],
   )
 
+  const { deleteOrders, deleting } = useDeleteOrders()
+
+  const onSubmit = useCallback(
+    async (event: React.SyntheticEvent<HTMLFormElement>): Promise<void> => {
+      event.preventDefault()
+
+      const success = await deleteOrders(Array.from(markedForDeletion.values()))
```

Comment:
No need for `Set.values()` here
```suggestion
      const success = await deleteOrders(Array.from(markedForDeletion))
```